### ISSUE TITLE
Bug: Empty Line Around Tables issue

### ISSUE THREAD

**Issue by shuojie819** [platers/obsidian-linter#713]
Sorry, my English is not good
Check out the video：（the reason is open "Empty Line Around Tables" ) 

https://github.com/platers/obsidian-linter/assets/72399630/6756ae02-bbf5-4fed-8f46-f7905e8de184

**Comment by pjkaufman** [comment]
@shuojie819 , could you provide the file contents for a reproducible example?

Google translated simplified Chinese:

@shuojie819 ，你能提供一个可重现的例子的文件内容吗？

**Comment by shuojie819** [comment]
> @shuojie819 , could you provide the file contents for a reproducible example?
> 
> Google translated simplified Chinese:
> 
> @shuojie819 ，你能提供一个可重现的例子的文件内容吗？


Example:

```md
> [!timeline]+ 事件记录
>
>> Left-hand column content
>
> ---
>
> Right column content

> [!timeline]+ 事件记录
>
>> 2000-07-19
>
> ---
>
> Right column content
>
> <br />
>
>> 1998-05-28
>
> Right column content
```

**Comment by pjkaufman** [comment]
I believe this is fixed on master and related to #704 . I will have to check later on.

Google translated simplified Chinese:
我相信这是固定在 master 上并且与 #704 相关。 我将不得不稍后检查。

**Comment by pjkaufman** [comment]
This is not an issue for me locally. I am assuming it was fixed by #704 . As such, I am closing this issue as a duplicate. Please let us know if this happens on master or in the next release.

Google translated simplified Chinese:
这对我本地来说不是问题。 我假设它已由 #704 修复。 因此，我将作为重复问题关闭此问题。 如果这发生在 master 或下一个版本中，请告诉我们。
